src/Components/Menu/Menu.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import Itens from './_itens';
+
+const drawerWidth = 240;
```

Comment:
Is This a really good place to put drawerWidth?

Reply:
In fact no, there's a new code using a setting.js file which contains the initial configurations.